content/tutorials/build-a-data-source-backend-plugin.md

```diff
@@ -0,0 +1,115 @@
+---
+title: Build a backend for a data source plugin
```

Comment:
I noticed that titles in this document are inconsistently cased. I.e. some are title cased, and some are not. What is our standard in this regard?

Reply:
This is documented in our [documentation style guide](https://github.com/grafana/grafana/blob/master/contribute/style-guides/documentation-style-guide.md#headings).

All headings should be sentence cased:

Yes: _Build a backend for a data source plugin_
No: _Build a Backend for a Data Source Plugin_

Reply:
Thanks @marcusolsson.